lib/defaults.js

```diff
@@ -44,13 +44,13 @@ module.exports = {
 
     // sets an array of middleware functions to handle login
     'login middleware': {
-        get: [require('../middleware-public/login').get],
-        post: [require('../middleware-public/authenticate')('linz-local', { failureFlash: true })]
+        get: [require('./api/middleware/login').get],
```

Comment:
I think it would be best to move the `require` statements to the top of this file?

Reply:
I've found moving modules to the top can cause circular dependencies where the module becomes `{}`. I haven't actually tried it here, so it might not change anything.
Did you want me to update it?

Reply:
@allanchau, let's try moving it to the top.